addons/modules/functions/fnc_bi_moduleCurator.sqf

```diff
@@ -0,0 +1,397 @@
+#include "script_component.hpp"
+/*
+ * Author: Bohemia Interactive
+ * Module function for initializing the Zeus Game Master module.
+ * Edited to allow for control over ascension messages and camera eagle,
+ * fix double attributes dialog bug, fix respawn on start with forced interface bug,
+ * and fix loss of curator assignment when the module's locality is transferred.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ * 1: Units <ARRAY>
+ * 2: Activated <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC, [], true] call zen_modules_fnc_bi_moduleCurator
+ *
+ * Public: No
+ */
+
+params ["_logic", "_units", "_activated"];
+
+if (_activated) then {
+    // BI's module framework is bugged and calls module function twice on clients after mission start
+    if (_logic getVariable [QGVAR(initialized), false]) exitWith {};
+
+    // Mark the module as initialized on this machine, fixes double attributes dialog bug
+    _logic setVariable [QGVAR(initialized), true];
+
+    // Exit if the module is not created on the server
+    if (!isServer && {local _logic} && {isNull getAssignedCuratorUnit _logic}) exitwith {
+        [format ["%1 is trying to create a Curator logic ModuleCurator_F.", profileName], "BIS_fnc_error", false] call BIS_fnc_MP;
```

Comment:
This is the same as:
```suggestion
        [format ["%1 is trying to create a Curator logic ModuleCurator_F.", profileName] remoteExec ["BIS_fnc_error", 2];
```
(it spawns the function like original, use `remoteExecCall` for unscheduled.)